### ISSUE TITLE
Feature Request - Add Alby Hub to Marketplace

### ISSUE THREAD

**Issue by greenbike** [mynodebtc/mynode#878]
**Describe the solution you'd like**
I would love to see Alby Hub added to the Marketplace

**Additional context**
To be able to create an lnurl that is self hosted and runs on MyNode and could work with NOSTR would be a great self-sovereignty feature.

- [ x ] Mark this if you searched the existing list of open/closed issue and did not find anything similar

**Comment by itstomekk** [comment]
Alby Hub doesn't come with LNURL service, but I'd love to see it on myNode as well - and many Alby users are asking for it.


It's already on marketplaces of [Umbrel](https://apps.umbrel.com/app/albyhub), [Start9](https://marketplace.start9.com/albyhub), [CasaOS](https://blog.getalby.com/alby-hub-brings-self-custodial-bitcoin-payments-to-casaos/), [Unraid](https://unraid.net/community/apps?q=alby#r), [RaspiBlitz](https://x.com/rootzoll/status/1871612490716688759)

**Comment by TheTotalConnector** [comment]
Adding Alby Hub to mynode apps (marketplace?) would make things for many users so much easier. 
especially when I am offline with my Alby Hub (right now via windows), I can still receive sats. 
Please, let me know. Thanks.

**Comment by mixpix** [comment]
Would really love to see this. I would not want to fix things if I manually installed it on mynode. They have a document on how to build an app for the platform here. https://mynodebtc.github.io/developer/add-app.html

**Comment by tlindi** [comment]
Working on it [here](https://github.com/mynodebtc/mynode/pull/902)

Likely v1.16 or later, and make it Community App first.

Need to implement automatic backup & restore similar to: https://github.com/tlindi/mynode-phoenixd/tree/main/scripts
or mynode-pwallet

**Comment by MoritzKa** [comment]
Alby Hub v1.17 is available https://github.com/getAlby/hub/releases

**Comment by tlindi** [comment]
I'm looking forward to complete PR data management ( [PR#940](https://github.com/mynodebtc/mynode/pull/940) support first and after that provide PR to update Alby Hub to latest at the moment (now v1.17.1) also.

I hope these are compeleted before a new MyNodeBTC version.

There would be a need for docs of setup Alby Hub with LND too.

**Comment by tehelsper** [comment]
Being added as part of the next release. Available in latest beta.

**Comment by MoritzKa** [comment]
There are two options to run Alby Hub:
- default with the integrated LDK node. That's safer for users because, they receive backup services automatically.
- install and connected to an external lightning node like LND

**Comment by tlindi** [comment]
> > There would be a need for docs of setup Alby Hub with LND too.
> 
> A guide or a way to auto-setup the config would be nice. All other apps automatically setup the LND config upon installation.

I asked help for such from Alby. But they refused and insisted their fancy intro must be presented to end user.

**Without** selecting "Advanced Setup" Alby Hub will setup internal/and additional LDK Lightning Node.

Therefore I see mandatory to have "Remove Data" before release. (For user to start over and learn to select "Advanced".

**Comment by tlindi** [comment]
> > There would be a need for docs of setup Alby Hub with LND too.
> 
> A guide or a way to auto-setup the config would be nice. >
That sure is needed.

**Comment by tlindi** [comment]
> There are two options to run Alby Hub:
> 
> * default with the integrated LDK node. That's safer for users because, they receive backup services automatically.

Yes, and it would create second Lightning Node for user, which I quess is rarely needed.

Safety issue of Lightning Channels is true. But MyNodeBTC already takes care of its LND (channel) backups on every change. Right @tehelsper ?

> * install and connected to an external lightning node like LND

As MyNode has LND and users likely have channels and sat capasity on it - would that Be Recommented Setup on MyNode.

**Comment by MoritzKa** [comment]
That's how [Start9](https://github.com/Start9Labs/albyhub-startos/blob/f79a40ee0d929e2554f1ee2cdef6c66fa2c00bdf/docker_entrypoint.sh#L1) and [Umbrel](https://github.com/getumbrel/umbrel-apps/tree/29fd6d1af5b6232f0de53a32677db94efafc39e6/albyhub) are doing it. 

I agree that LND is the right way to start for existing MyNode LND users. 
However, Alby Hub + integrated LDK is the safer and more convenient way to start for new users, because they receive dynamic channel backups without having to close channels to recover funds. Additionally Alby Hub + LDK is a lot more resource efficient for the underlying MyNode hardware.

**Comment by tlindi** [comment]
> That's how [Start9](https://github.com/Start9Labs/albyhub-startos/blob/f79a40ee0d929e2554f1ee2cdef6c66fa2c00bdf/docker_entrypoint.sh#L1) and [Umbrel](https://github.com/getumbrel/umbrel-apps/tree/29fd6d1af5b6232f0de53a32677db94efafc39e6/albyhub) are doing it.
> 
> I agree that LND is the right way to start for existing MyNode LND users. However, Alby Hub + integrated LDK is the safer and more convenient way to start for new users, because they receive dynamic channel backups without having to close channels to recover funds. Additionally Alby Hub + LDK is a lot more resource efficient for the underlying MyNode hardware.

I see you're saying MyNodeBTC should provide guide to help users to manually connect to local LND, and no automatic / API based setup is available - people just need to see Intro and paste HEX TSL etc to Advanced Setup Screen - right?

**Comment by tehelsper** [comment]
Yeah, MyNode will backup the channel file and it may be very confusing to have two LND wallets. A guide would be helpful.
